modules/dxp/apps/portal-workflow/portal-workflow-metrics-rest-test/src/testIntegration/java/com/liferay/portal/workflow/metrics/rest/resource/v1_0/test/helper/WorkflowMetricsRESTTestHelper.java

```diff
@@ -1009,15 +1028,36 @@ public void deleteTasks(long companyId, long processId) throws Exception {
 	}
 
 	public void restoreProcess(Document document) throws Exception {
+		AddProcessRequest.Builder builder = new AddProcessRequest.Builder();
+		String[] versions = {
+			StringBundler.concat(
+				document.getString("version"), CharPool.PERIOD, 0)
+		};
```

Comment:
Hi @seltong,

What do you think this approach below:

`AddProcessRequest.Builder builder = new AddProcessRequest.Builder();`

`builder.active ... titleMap( ...`

`String version = StringBundler.concat(document.getString("version"), CharPool.PERIOD, 0);`

`builder.version(version).versions(new String[] { version })`

`_processWorkflowMetricsIndexer.addProcess(builder.build());`

slack me monday

Reply:
Just to bring the version variable more closer to its use

Reply:
Likes good for me. I talk to you about it on Monday